### ISSUE TITLE
Images schema should contain description, credit, and creditUrl.

### ISSUE THREAD

**Issue by bobclewell** [apostrophecms/apostrophe#980]
It is often desired that the “description” field associated with an image is used for as an “alt” property in an HTML’s img tag. It would be great if `image.description` would return this data. Currently the description (and credit, and creditUrl) is not part of the image schema, so something like this  `widget.image.items[0]._pieces[0].item.description` is required to get access to that data.

A sample of the current schema:
```
{ _id: 'cj4twxvb0007yj2gzj3x5d5yl',
  crop: null,
  length: null,
  group: 'images',
  createdAt: Fri Jun 07 2017 09:44:55 GMT-0400 (EDT),
  name: 'jim-bio-photo',
  title: 'jim bio photo',
  extension: 'jpg',
  type: 'attachment',
  md5: 'e9b9d330e5e0ade12d645ee780ef84e0',
  width: 600,
  height: 800,
  landscape: true,
  ownerId: 'cj4jzne3s0007apmc2t3uqa04',
  used: true,
  _crop: { left: 39, top: 0, width: 600, height: 600 } }
```

**Comment by boutell** [comment]
The issue is that the first helper returns the attachment field, not the
entire piece. This is useful because that's what you need to get a URL. We
will need to talk through ways to access the containing piece too. Possibly
a _doc property.

**Comment by stuartromanek** [comment]
+1 on making these properties easier to access

**Comment by mtthwmnc** [comment]
This was merged in #1600. Closing this issue.
